packages/common/src/lib/common/tools.ts

```diff
@@ -1269,16 +1314,16 @@ async function getRepositoryFileAsync(url, hash, force, _actualRepo) {
     let data;
 
     if (url.startsWith('http://') || url.startsWith('https://')) {
-        axios = axios || require('axios');
         if (!_hash) {
             _hash = await axios({ url: url.replace(/\.json$/, '-hash.json'), timeout: 10000 });
         }
 
         if (_actualRepo && hash && _hash && _hash.data && _hash.data.hash === hash) {
             data = _actualRepo;
         } else {
-            const agent = `${module.exports.appName}, RND: ${randomID}, Node:${process.version}, V:${
-                require('../../package.json').version
+            const agent = `${exports.default.appName}, RND: ${randomID}, Node:${process.version}, V:${
+                // eslint-disable-next-line @typescript-eslint/no-var-requires
+                require('../../../package.json').version
```

Comment:
We should require the actual module name `@iobroker/js-controller-common/package.json` instead of relying on relative paths.